src/KubernetesClient/KubernetesClientConfiguration.ConfigFile.cs

```diff
@@ -28,6 +29,11 @@ public partial class KubernetesClientConfiguration
         // For testing
         internal static string KubeConfigEnvironmentVariable { get; set; } = "KUBECONFIG";
 
+        /// <summary>
+        /// Exec process timeout in seconds
+        /// </summary>
+        public static int ExecTimeout { get; set; } = 10;
```

Comment:
any reason why this is not per inst var?

Reply:
@tg123 currently the ExecuteExternalCommand command which uses it is Static.

Reply:
oh i see, what about make ExecuteExternalCommand non static?
it is weird to this value static

Reply:
Its static as its used outside here, https://github.com/kubernetes-client/csharp/blob/e42290d6f7c7fbb515cca119e8c3f7507bd00ce2/src/KubernetesClient/Authentication/ExecTokenProvider.cs#L46

Reply:
oh i see, that is difficult to change, lets keep it `static`

ps i do not think anyone is going to modify it